com.unity.render-pipelines.high-definition/Documentation~/Dynamic-Resolution.md

```diff
@@ -76,39 +82,47 @@ public class DynamicRes : MonoBehaviour
+| FidelityFX Super Resolution 1.0      | FidelityFX Super Resolution 1.0 uses a spatial super-resolution method that balances quality and performance. For more information, see [AMD FidelityFX](https://www.amd.com/en/technologies/radeon-software-fidelityfx).<br/><br/> FidelityFX Super Resolution 1.0 has no dependencies and runs at the end of the post-processing pipeline.<br />FidelityFX Super Resolution 1.0also runs when at 100% resolution as it can have beneficial sharpening effects. |
+| Temporal Anti-Aliasing (TAA) Upscale | Temporal Anti-Aliasing (TAA) Upscale uses temporal integration to produce a sharp image. Unity performs this method alongside the normal anti-aliasing to optimise performance.<br />HDRP executes this upscale filter before post processing and at the same time as the TAA step. This means you can only use the TAA anti-aliasing method. This filter is not compatible with other anti-aliasing methods. <br /><br/>Temporal Anti-Aliasing (TAA) Upscale performs antialiasing on each frame. This means that it also runs when you enable Dynamic Resolution, even when the screen percentage is at 100% resolution. <br />For more information, see the section [Notes on TAA Upscale](Dynamic-Resolution.md#Notes). |
+
+## Overriding upscale options with code
+
+You can also override the upscale options in the HDRP Asset for each Camera in your scene using code. To do this, call DynamicResolutionHandler.SetUpscaleFilter(Camera camera, DynamicResUpscaleFilter filter), to make HDRP ignore the value in the HDRP Asset for a given Camera.
 
-You can also override the upscale  options in the HDRP asset for each Camera in your scene using code.  To do this, call `DynamicResolutionHandler.SetUpscaleFilter(Camera camera, DynamicResUpscaleFilter filter)`, to make HDRP ignore the value in the HDRP Asset for a given camera.
+<a name="TAA_Notes"></a>
 
 ## Notes on Temporal Anti-Aliasing (TAA) Upscaling
 
-When you enable Temporal Anti-Aliasing (TAA) Upscaling, it replaces Temporal Anti-Aliasing in the pipeline. This means:
+When you enable **Temporal Anti-Aliasing (TAA) Upscaling**, it replaces Temporal Anti-Aliasing in the pipeline. This means:
 
-- Temporal Anti-Aliasing is the only the anti-aliasing method and no other post-process anti-aliasing option will work.
-- The post-process pipeline runs at final resolution. Before HDRP applies TAA Upscaling, the pipeline uses a downscaled resolution. This gives a more precise post-processing result, but also uses more GPU resources.
+- When TAA Upscaling is active, it is the only anti-aliasing method. No other post-process anti-aliasing option will work.
```

Comment:
This is not what I meant with the original phrase, I actually meant TAA and not TAAU

Reply:
Sorry being a bit confusing here, but it is when TAAU is active, TAA (without the U :D ) is the only anti-aliasing method.